src/handlers/inviteLink.ts

```diff
@@ -0,0 +1,126 @@
+                    author: {
+                        name: msg.author.tag,
+                        iconURL: msg.author.displayAvatarURL({ dynamic: true }),
+                    },
+                    fields: [
+                        {
+                            name: 'User:',
+                            value: `${msg.author}`,
+                        },
+                        {
+                            name: 'Channel:',
+                            value: `${msg.channel}`,
+                        },
+                        {
+                            name: 'Message Content:',
+                            value: codeBlock(msg.content.toString()),
+                        },
+                    ],
+                });
+
+                await modLogs.send({ embeds: [allowedEmbed] });
+                return;
+            }
+
+            await msg.delete();
+
+            const notAllowedEmbed = makeEmbed({
+                title: 'Invite Link Detected',
+                thumbnail: { url: 'https://cdn.discordapp.com/attachments/932350968522240101/932625893657026630/Scam.png' },
+                author: {
+                    name: msg.author.tag,
+                    iconURL: msg.author.displayAvatarURL({ dynamic: true }),
+                },
+                fields: [
+                    {
+                        name: 'User:',
+                        value: `${msg.author}`,
+                    },
+                    {
+                        name: 'Channel:',
+                        value: `${msg.channel}`,
+                    },
+                    {
+                        name: 'Message Content:',
+                        value: codeBlock(msg.content.toString()),
+                    },
+                ],
+            });
+            try {
+                await msg.member.timeout(60 * 60 * 24 * 1000, 'Invite Link');
+            } catch (e) {
+                Logger.error(e);
+                const errorEmbed = makeEmbed({
+                    title: 'Error timing out user',
+                    description: makeLines([
+                        `An error occurred while timing out ${msg.author}`,
+                        `${codeBlock(`Error : ${e}`)}`,
+                    ]),
+                    color: Colors.Red,
+                });
+                await modLogs.send({ embeds: [errorEmbed] });
+
+                await modLogs.send({ embeds: [notAllowedEmbed] });
+                return;
+            }
+
+            try {
+                await msg.author.send('You sent a Discord server invite in the FlyByWire Simulations Discord server. This function is in place to prevent discord scams/unwanted spamming of invites and has resulted in an automatic seven day timeout and notification of our moderation team. If this was done in error, our moderation team will reverse the timeout, however please refrain from sending invites in the future. Specific invites can be sent through the FlyByWire Bot.');
```

Comment:
Depending on the timer as per above, might need to change `seven day` to `one day` or `seven days`